Core/Relationships/RelationshipResolver.cs

```diff
@@ -396,10 +457,10 @@ public List<CkanModule> ModList()
 
         public bool IsConsistant
         {
-            get { return conflicts.Any(); }
+            get { return !conflicts.Any(); }
```

Comment:
Oh my!

I'm also very confused. The only place this seems to be used is in our test suite, which I would expect would have spotted this doing the reverse of what was intended.